src/ui/player_bar.py

```diff
@@ -14,6 +14,8 @@ def get_experience():
     wait(0.05)
     # crop roi
     img = grab()
+    
+    mouse.move(x_m, y_m-50, randomize = (8,1))
```

Comment:
I don't believe grab() includes the mouse, does it? Never seen the mouse in any screenshots from cv

Reply:
Added this so that the mouse goes back into the main area as the next action Botty does is try and tp back to town. So figured would just put the mouse back into the game area. 

Otherwise Botty would try and right click tp,  but mouse is over xp bar and tp isn't made, Botty waits until the timeout and repositions to tp again. 

Other thought was have the tp process make sure the mouse is on screen somewhere clickable but this was easier haha